test/test_other.py

```diff
@@ -7154,7 +7155,7 @@ def test_realpath_nodefs(self):
 }
 ''')
     create_file('TEST_NODEFS.txt', ' ')
-    self.run_process([EMCC, 'src.c', '-lnodefs.js'])
+    self.emcc('src.c', args=['-lnodefs.js', '-sFORCE_FILESYSTEM'])
     self.assertContained('Resolved: /working/TEST_NODEFS.txt', self.run_js('a.out.js'))
```

Comment:
Can you instead convert both these two lines into a single call to `self.do_runf` ?

Reply:
Done.